### DISCUSSION TITLE
WPFHexaEditor.dll got flagged by antivirus

### DISCUSSION THREAD

**Discussion by Filipsi** [abbaye/WPFHexEditorControl#126] [Q&A] [answered]
I am using this control in my application and users that have ESET antivirus reported that the application stopped working.
According to the logs, ESET removed WPFHexaEditor.dll from application folder due to it been possible variant of [MSIL/Kryptik.AFVX](https://www.virusradar.com/en/threat_encyclopaedia/detail/498762).

It seems that the dll for net47 got [flagged](https://www.virustotal.com/gui/file/29db059e25e4b72abecadbdb66bc7daa8069cfb152e441beee7d6f990fb8bb6b/detection) as virus 3 days ago.

As it stands, users can't use any app with this control, without turning off their antivirus protection.
Is this a known issue?

**Comment by abbaye** [comment]
Hello,

I think is an error of ESET antivirus. 🤷

Check the code on GitHub and build your own copy. My code is safe and clean
of anything.

Best regards

Le lun. 1 août 2022, 7 h 34 a.m., Filip Šikula ***@***.***> a
écrit :

**Comment by Filipsi** [comment, marked as answer]
I have submitted a sample to ESET and they confirmed a false positive.
This issue should be fixed in the next update of ESET detection engine.

**Comment by abbaye** [comment]
Nice, thank you mister

Le mer. 3 août 2022, 1 h 58 a.m., Filip Šikula ***@***.***> a
écrit :
